.github/workflows/ci.yml

```diff
@@ -34,6 +34,7 @@ jobs:
           make
 
       - name: flake8
+        if: always()
```

Comment:
Just curious, what does this line do?

Reply:
Actually I think the style check of the CI should be changed to:

```
steps:
      - uses: actions/checkout@v2

      - name: Setup Python
        uses: actions/setup-python@v2
        with:
          python-version: 3.8

      - name: Install style requirements
        run: |
          pip install -r requirements_style.txt --disable-pip-version-check

      - name: Codespell
        run: |
          make codespell

      - name: flake8
        run: |
          make flake8
```

Reply:
It seems he wants to run flake8 no matter what...

https://github.community/t/multiple-if-conditional-with-always-is-always-true/128840

Reply:
@jleonatti, do you think we could add `if: always()`?   We'd add it to:

https://github.com/pyansys/PyAEDT/blob/f6a91ddb224aabc5950400002575af342f813372/.github/workflows/style.yml#L26-L28